docs/api.rst

```diff
@@ -50,8 +50,12 @@ reference for developers, not a gospel.
     api/camayoc.tests.qcs.ui.views
     api/camayoc.tests.qcs.utils
     api/camayoc.tests.remote
+    api/camayoc.tests.remote.conftest
```

Comment:
Did you use the gen_api_docs.sh script? Just checking if it is really working :smile:

Reply:
I ran `make all`, so it looks like it ran!